docs/src/api.md

```diff
@@ -40,6 +40,7 @@ RaceFilterPersonIDs
 GenderFilterPersonIDs
 StateFilterPersonIDs
 AgeGroupFilterPersonIDs
+GetVisitProcedure
```

Comment:
Could you please put this over in the "Getters" section? But thanks for adding this in!